### ISSUE TITLE
full code multi label classfication

### ISSUE THREAD

**Issue by ghaith-kh** [kaushaltrivedi/fast-bert#37]
hey, can you please provide me of the full code of multi-label using xlnet on the toxic data set,
i will be so grateful

**Comment by amin-nejad** [comment]
You can find a full working example in the `sample_notebooks` folder

**Comment by ghaith-kh** [comment]
Hey bro i have data set like the toxic which contain 5 labels and im
looking for a way to label it automatically like putting 0 and 1 in every
class label , do u know a way to do it , please reply me im in crucial time
😞

**Comment by amin-nejad** [comment]
Sorry mate, I can't help you if I don't know what the problem is. Just follow the instructions in the README, or the [medium article](https://medium.com/huggingface/introducing-fastbert-a-simple-deep-learning-library-for-bert-models-89ff763ad384). Works fine for me

**Comment by ghaith-kh** [comment]
Yes works fine for me also i just asked you if you know auto labeling code
that can help me to label a reviews data set.
